gateway-service/src/main/java/org/zowe/apiml/gateway/filters/post/ConvertAuthTokenInUriToCookieFilter.java

```diff
@@ -9,69 +9,66 @@
+        RequestContext context = RequestContext.getCurrentContext();
+        AuthConfigurationProperties.CookieProperties cp = authConfigurationProperties.getCookieProperties();
+        return context.getRequestQueryParams() != null && context.getRequestQueryParams().containsKey(cp.getCookieName());
     }
 
     public Object run() {
         RequestContext context = RequestContext.getCurrentContext();
-        AuthConfigurationProperties.CookieProperties cp = authConfigurationProperties.getCookieProperties();
-        if ((context.getRequestQueryParams() != null) && context.getRequestQueryParams().containsKey(cp.getCookieName())) {
-            HttpServletResponse servletResponse = context.getResponse();
+        HttpServletResponse servletResponse = context.getResponse();
 
-            // SameSite attribute is not supported in Cookie used in HttpServletResponse.addCookie,
-            // so specify Set-Cookie header directly
-            String cookieHeader = CookieUtil.setCookieHeader(
-                cp.getCookieName(),
-                context.getRequestQueryParams().get(cp.getCookieName()).get(0),
-                cp.getCookieComment(),
-                cp.getCookiePath(),
-                cp.getCookieSameSite().getValue(),
-                cp.getCookieMaxAge(),
-                true,
-                true
-            );
-            servletResponse.addHeader("Set-Cookie", cookieHeader);
+        // SameSite attribute is not supported in Cookie used in HttpServletResponse.addCookie,
+        // so specify Set-Cookie header directly
+        String cookieHeader = CookieUtil.setCookieHeader(
+            cp().getCookieName(),
+            context.getRequestQueryParams().get(cp().getCookieName()).get(0),
+            cp().getCookieComment(),
+            cp().getCookiePath(),
+            cp().getCookieSameSite().getValue(),
+            cp().getCookieMaxAge(),
+            true,
+            true
+        );
+        servletResponse.addHeader("Set-Cookie", cookieHeader);
 
-            String url = context.getRequest().getRequestURL().toString();
-            String newUrl;
-            if (url.endsWith("/apicatalog/")) {
-                newUrl = url + "#/dashboard";
-            } else {
-                newUrl = url;
-            }
-            context.addZuulResponseHeader("Location", newUrl);
-            context.setResponseStatusCode(HttpServletResponse.SC_MOVED_TEMPORARILY);
+        String url = context.getRequest().getRequestURL().toString();
+        String newUrl;
+        if (url.endsWith("/apicatalog/")) {
+            newUrl = url + "#/dashboard";
+        } else {
+            newUrl = url;
         }
+        context.addZuulResponseHeader("Location", newUrl);
+        context.setResponseStatusCode(HttpServletResponse.SC_MOVED_TEMPORARILY);
         return null;
     }
+
+    private AuthConfigurationProperties.CookieProperties cp()  {
+        return authConfigurationProperties.getCookieProperties();
+    }
```

Comment:
Does it make sense to make a method call here every time? vs. Just getting the properties once in the method and then referencing the CookieProperties object directly each time.